<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dean_menu</class>
 <widget class="QWidget" name="dean_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_v_x1">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>104</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nu_De">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>137</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>µ = </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_De_y1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>216</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_v">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>104</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>v =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_p_x1">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>38</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nu_De_x1">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>137</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_D">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>71</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_De">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>220</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>De Y1 = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>170</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>r = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_D_x1">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>71</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_p">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>38</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>p =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_r_x1">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>170</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Value_delete">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>360</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить значения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_parametrs_to_the_corresponding_graph_Dean">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>300</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить параметры в 
 соответствующий 
  график</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>38</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_p_x2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>38</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_r_x2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_De_y2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>256</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_D_x2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>71</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_v_x2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>104</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nu_De_x2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>137</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите число от 1000,00 до 5000,00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>71</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>104</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>137</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>170</y>
     <width>16</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_X1">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_X2">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_De_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>260</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>De Y2 = </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
